compiler/damlc/stable-packages/src/GenerateStablePackage.hs

```diff
@@ -497,6 +499,34 @@ daInternalDown = package version1_6 $ NM.singleton Module
       [ mkWorkerDef modName downTyCon tyVars [(unpackField, TVar tyVar)]
       ]
 
+daInternalException :: Package
+daInternalException = Package
+    { packageLfVersion = versionDev -- TODO (#8020): Update LF version when we stabilize exceptions.
+    , packageMetadata = Just PackageMetadata
+        { packageName = PackageName "stable-daml-stdlib-DA-Internal-Exception"
+        , packageVersion = PackageVersion "1.0.0"
+        }
+    , packageModules = NM.singleton Module
+        { moduleName = modName
+        , moduleSource = Nothing
+        , moduleFeatureFlags = daml12FeatureFlags
+        , moduleSynonyms = NM.empty
+        , moduleDataTypes = types
+        , moduleValues = values
+        , moduleTemplates = NM.empty
+        }
+    }
+  where
+    modName = mkModName ["DA", "Internal", "Exception"]
```

Comment:
You’ll have to extend the tests that check that we don’t accidentally modify a stable package. CI should hopefully find that :crossed_fingers:

Reply:
Hmm ... Where are these tests? I don't think they've triggered.

Reply:
Ok, after looking into this with @cocreature, the reason this didn't trigger is because we don't run the stable package test in `1.dev`. And that's ok! We shouldn't run it in 1.dev since whatever "stable package" we have in 1.dev isn't actually stable yet :-)

Once we stabilize exceptions, we'll have to add the new stable package to this test. I added this in the issue #8020.